src/modules/shared/components/Blade/Blade.tsx

```diff
@@ -69,7 +69,10 @@ const Blade: FC<BladeProps> = ({
 				className={clsx(
 					className,
 					styles['c-blade'],
-					isBladeOpen && styles['c-blade--visible']
+					isBladeOpen && styles['c-blade--visible'],
+					{
+						[styles['c-blade--managed']]: isLayered,
+					}
```

Comment:
Gebruik je hier niet 2 notaties door elkaar?
```suggestion
					isBladeOpen && styles['c-blade--visible'],
					isLayered && styles['c-blade--managed'],
```

Of komt dit niet op hetzelfde neer? Ik heb nog nooit met clsx gewerkt.

Reply:
Goed gezien! my bad.